.changeset/fuzzy-bees-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"blitz": patch
+---
+
+APP_ENV wasn't being set when using the -e flag. This fixes it.
```

Comment:
Can you rephrase it? It will be in in release notes, and it's not clear what "this" in the second sentence refers to.